source/includes/_intents.md

```diff
@@ -41,22 +41,102 @@ Please do not share your access tokens with other users or applications and don'
+**conversationId** | string | false | An ID used by the chat provider to identify the conversation.<br />The chat provider can decide the format.
 
 ## Response
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris scelerisque magna eu 
-sapien rutrum euismod. Praesent est urna, hendrerit vitae luctus non, mollis non odio. 
-Suspendisse potenti. Pellentesque pharetra ultrices lectus, sit amet accumsan justo 
-pellentesque pretium. Quisque et mattis leo, ut elementum magna. Praesent at placerat 
-quam. Ut tellus quam, consectetur quis metus sit amet, fermentum suscipit tortor. 
-Donec elit elit, pellentesque a turpis a, porttitor auctor metus. Fusce non rhoncus 
-erat, quis placerat tortor. Morbi molestie porta purus et consequat. Aliquam laoreet 
-at leo fermentum ornare. Fusce ut lobortis tortor, in tincidunt purus.
+> Example Response:
+
+```json
+{
+    "intents": [
+        {
+            "confidence": 0.64636783599853516,
+            "name": "intent A"
+        },
+        {
+            "confidence": 0.54660784006118774,
+            "name": "intent B"
+        },
+        {
+            "confidence": 0.34867138862609863,
+            "name": "intent C"
+        },
+        {
+            "confidence": 0.1374831646680832,
+            "name": "intent D"
+        },
+        {
+            "confidence": 0.1208697184920311,
+            "name": "intent E"
+        }
+    ],
+    "entities": [
+        {
+            "confidence": 0.7,
+            "form": "name.surname@mail.com",
+            "index": 0,
+            "name": "email",
+            "id": "610abae77bf221077b4fc51e",
+            "sanitize": true
+        }
+    ]
+}
+```
+
+The endpoint will respond with a JSON with the following format:
+
+Field | Type  | Description
+---------- | ---------- | ------------------------------
+**intents** | array[`Intent`] | Array containing intents that were predicted for the `message` sent in the request.<br /><br />Each object of type `Intent` contains the following fields:<br />**name** `string` - the intent's name<br />**confidence** `float` - Float value between 0 and 1 - giving the confidence with which this intent was predicted for the given `message`.
+**entities** | array [`Entity`] |  Array of entities that were recognized from the visitor message given in the request.<br /><br />Each object of type `Entity` will have the following fields:<br />**name** `string` - Name of the recognized entity<br />**confidence** `float` - Float value between 0 and 1 - giving the confidence with which this entity was recognized for the given `message`.<br />**form** `string` - Form of the entity recognized in the given `message`.<br />**index** `number` - TODO Index of the entity recognized.<br />**id** `number` - Identifier of the entity recognized.<br />**sanitize** `boolean` - Boolean marking if entity is sanitized.
+
+## Usage
+### Start a conversation
```

Comment:
there's no starting of a conversation here, so i'd just stick to "usage" without a sub section

Reply:
copy-paste god, forgive me!